translator-files/po/de/Server-Linux.po

```diff
@@ -227,7 +227,7 @@ msgstr ""
 
 #. type: Plain text
 #: wiki/en/Server-Linux.md:63
-msgid "`journalctl -u jamulus | grep connected`"
+msgid "`journalctl -u jamulus-headless | grep connected`"
```

Comment:
Something went wrong here. It should have been up to date already...

Reply:
Nothing wrong - this doc hasn't been translated into German. What's been updated is the source English segment. I directly edited the .po files when this came up, and only did the docs that had been translated + EN.